tests/run/legacy_implicit_noexcept_build.srctree

```diff
@@ -31,3 +32,16 @@ except RuntimeError:
     pass
 else:
     assert False, 'Exception not raised'
+
+
+######## aa.pxd ########
+cdef class Klass:
+    cpdef int m(self) except? -1
+
```

Comment:
```suggestion
    cpdef int m2(self)
    cpdef int m3(self) noexcept
    cpdef int m4(self) except *
    cpdef int m5(self) except -1
    cpdef object n1(self)
    cpdef object n2(self) except *
    cpdef object n3(self) noexcept  # will warn, but should work
    cpdef void o1(self)
    cpdef void o2(self) except*
    cpdef void o3(self) noexcept
    cdef int p1(self) except? -1
    cdef int p2(self)
    cdef int p3(self) noexcept
    cdef int p4(self) except *
    cdef int p5(self) except -1
    cdef object q1(self)
    cdef object q2(self) except *
    cdef object q3(self) noexcept  # will warn, but should work
    cdef void r1(self)
    cdef void r2(self) except*
    cdef void r3(self) noexcept

cpdef int m1(self) except? -1
cpdef int m2()
cpdef int m3() noexcept
cpdef int m4() except *
cpdef int m5() except -1
cpdef object n1()
cpdef object n2() except *
cpdef object n3() noexcept  # will warn, but should work
cpdef void o1()
cpdef void o2() except*
cpdef void o3() noexcept
cdef int p1() except? -1
cdef int p2()
cdef int p3() noexcept
cdef int p4() except *
cdef int p5() except -1
cdef object q1()
cdef object q2() except *
cdef object q3() noexcept  # will warn, but should work
cdef void r1()
cdef void r2() except*
cdef void r3() noexcept
```

Since this is obviously a little under-tested, just just suggesting adding a long list of similar examples here in the hope that it catches any other edge cases

Reply:
Fixed. I think I should also implement tests for exception propagation (non-propagation of noexcept functions) for every method/function I suppose...

Reply:
>  think I should also implement tests for exception propagation

done